### ISSUE TITLE
Newline flag removes right modules

### ISSUE THREAD

**Issue by polothy** [justjanne/powerline-go#143]
Using `v1.13.0` with this config:
```zsh
function powerline_precmd() {
    eval "$($GOPATH/bin/powerline-go -error $? -shell zsh -eval -newline -modules 'venv,cwd,perms,git,jobs,exit,root,vgo' -modules-right 'aws')"
}

function install_powerline_precmd() {
    for s in "${precmd_functions[@]}"; do
        if [ "$s" = "powerline_precmd" ]; then
            return
        fi
    done
    precmd_functions+=(powerline_precmd)
}

if [ "$TERM" != "linux" ]; then
    install_powerline_precmd
fi
```

With the ` -newline` flag, right modules no longer displays.  Without ` -newline`, then right modules do display.

**Comment by artsilenkov** [comment]
Same for me, -newline flag removes right modules at all

**Comment by justjanne** [comment]
Newline on its own line is not really possible with the current implementation of right modules, sadly. This would require a major rework

**Comment by wjam** [comment]
Note that the workaround for this is to use the `newline` and `root` modules like:
`powerline-go -colorize-hostname -eval -modules 'user,host,cwd,newline,root' -modules-right 'time' -shell zsh`

**Comment by zoj613** [comment]
> Note that the workaround for this is to use the `newline` and `root` modules like:
> `powerline-go -colorize-hostname -eval -modules 'user,host,cwd,newline,root' -modules-right 'time' -shell zsh`

does this work with bash?

**Comment by duckpuppy** [comment]
@wjam Does that actually work for you? Your exact command for me does not actually include a newline - there's a visual space between the CWD segment and the root segment, but they're on the same line.

**Comment by wjam** [comment]
Yes, mine works. My [dotfiles repo](https://github.com/wjam/dotfiles) contains my current configuration which has a [ZSH theme](https://github.com/wjam/dotfiles/blob/master/zsh-themes/iterm2-powerline-go.zsh-theme) that makes use of [JSON config file for powerline-go](https://github.com/wjam/dotfiles/blob/master/config/powerline-go/config.json).

**Comment by duckpuppy** [comment]
For me, if I use the `--eval` option, newline doesn't work at all... it just inserts a space where the newline should be.
